feature/funding/googleplay/src/main/kotlin/app/k9mail/feature/funding/googleplay/GooglePlayFundingManager.kt

```diff
@@ -1,10 +1,19 @@
 package app.k9mail.feature.funding.googleplay
 
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
 import app.k9mail.feature.funding.api.FundingManager
 import app.k9mail.feature.funding.api.FundingType
+import app.k9mail.feature.funding.googleplay.ui.reminder.FundingReminderContract
 
-class GooglePlayFundingManager : FundingManager {
+class GooglePlayFundingManager(
+    private val reminder: FundingReminderContract.Reminder,
+) : FundingManager {
     override fun getFundingType(): FundingType {
         return FundingType.GOOGLE_PLAY
     }
+
+    override fun addFundingReminder(activity: AppCompatActivity, launcherIntent: Intent) {
```

Comment:
This uses `launcherIntent` instead of the `launcherBaseIntent` that is used in the interface. This leads to a warning in the IDE ("This may cause problems when calling this function with named arguments").